packages/server/src/tests/jestEnv.ts

```diff
@@ -3,6 +3,7 @@ import { tmpdir } from "os"
 process.env.SELF_HOSTED = "1"
 process.env.NODE_ENV = "jest"
 process.env.MULTI_TENANCY = "1"
+process.env.APP_PORT = "0"
```

Comment:
should we do the same for the worker - just to avoid the issue that you ran into?